app/config.json

```diff
@@ -1,3 +1,3 @@
 {
-  "serviceName": "Manage free school projects prototype"
```

Comment:
@adyhoran1 are you sure you want to remove the word "prototype" here? Are we not worrying about misleading users?

Reply:
Thanks @mingoland, but I'm not worried - Zuleima is good at making it clear to users in research sessions that it is a prototype. Plus removing 'prototype' from the name:

- allows us to test further how users feel about the name Manage free school projects
- is consistent with the naming convention for other RSD products, like [Prepare](https://prepare-conversions-transfers.herokuapp.com/) and [Complete](https://conversions-prototype.herokuapp.com/0-7/start)

Reply:
Alright in this case let's just remove the word "prototype" then. 

Other changes look fine to me too. Just want to remind you that the changes you made on the task list will be reflected to all projects. We can still make tasks with different statuses for different projects. We can work on it together. 

Happy for you to merge this branch.